### ISSUE TITLE
Selected campaign item colour doesn't meet accessibility standards

### ISSUE THREAD

**Issue by newleeland** [silverstripe/silverstripe-campaign-admin#130]
## Description

![Screen Shot 2019-01-18 at 3.45.30 PM.png](https://images.zenhubusercontent.com/596d365561b4ca3b0a74bd86/c083e676-78ce-4b08-9a12-e58259f86f69)
Light blue background and white text does not meed WCAG AA standards.
Should be using "selected" variable (#0071c4)
![Screen Shot 2019-01-18 at 3.48.44 PM.png](https://images.zenhubusercontent.com/596d365561b4ca3b0a74bd86/5ff158e8-0339-492e-aa0e-1c1497626eb1)

State tag colour address in another story #77 

## Affected versions
SS 4.x

## Pull requests

- [x] https://github.com/silverstripe/silverstripe-admin/pull/816

**Comment by ScopeyNZ** [comment]
Closed in silverstripe/silverstripe-admin#816
